### PR TITLE
Show liquidation risk acknowledgement in dialogs

### PR DESCRIPTION
**Pull request by oskarvu**
Closes FRO-500

### CODE REVIEW

File: packages/app/src/domain/common/risk.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ import BigNumber from 'bignumber.js'
 export type RiskLevel = 'liquidation' | 'risky' | 'moderate' | 'healthy' | 'unknown' | 'no debt'
 
 export const LIQUIDATION_HEALTH_FACTOR_THRESHOLD = new BigNumber(1)
+export const LIQUIDATION_DANGER_HEALTH_FACTOR_THRESHOLD = new BigNumber(1.5)
```
Comment (yivlad) [inline packages/app/src/domain/common/risk.ts:6]: Is Aave using this number as well? Seems pretty high to me

File: packages/app/src/domain/liquidation-risk-warning/useLiquidationRiskWarning.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  updatedHealthFactor: BigNumber | undefined,
+  currentHealthFactor: BigNumber | undefined,
+  isFormValid: boolean,
+): boolean {
+  if (!updatedHealthFactor || !isFormValid) {
+    return false
+  }
+
+  const isUpdatedHFInDangerZone = updatedHealthFactor.lt(LIQUIDATION_DANGER_HEALTH_FACTOR_THRESHOLD)
+  // @note This check applies when a user attempts to withdraw disabled collateral. Even if position in the danger zone, no warning should be displayed.
```
Comment (krzkaczor) [inline packages/app/src/domain/liquidation-risk-warning/useLiquidationRiskWarning.ts:44]: good catch 👍

File: packages/app/src/domain/liquidation-risk-warning/useLiquidationRiskWarning.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  const hasHFChanged = !currentHealthFactor?.eq(updatedHealthFactor)
+  return hasHFChanged && isUpdatedHFInDangerZone
```
Comment (krzkaczor) [inline packages/app/src/domain/liquidation-risk-warning/useLiquidationRiskWarning.ts:46]: [Highway to the danger zone](https://www.youtube.com/watch?v=kyAn3fSs8_A)

File: packages/app/src/ui/organisms/risk-acknowledgement/RiskAcknowledgement.tsx [hunk 4]
```diff
@@ -36,6 +23,14 @@ export function RiskAcknowledgement({ warning, onStatusChange }: RiskAcknowledge
               warning.discrepancy,
               { style: 'auto' },
             )} ${warning.token.symbol}.`}
+          {warning.type === 'liquidation-warning-borrow' &&
+            'Borrowing this amount puts you at risk of quick liquidation. You may lose all of your collateral.'}
```
Comment (yivlad) [inline packages/app/src/ui/organisms/risk-acknowledgement/RiskAcknowledgement.tsx:27]: I guess it's actually `you may lose part of your collateral`?

**Review by krzkaczor — APPROVED** [review]

File: packages/app/src/domain/liquidation-risk-warning/useLiquidationRiskWarning.ts (hunk 3 above)
Comment (oskarvu) [reply packages/app/src/domain/liquidation-risk-warning/useLiquidationRiskWarning.ts:46]: im-ready-lets-go-gif

File: packages/app/src/domain/common/risk.ts (hunk 1 above)
Comment (oskarvu) [reply packages/app/src/domain/common/risk.ts:6]: I set this for `1.2` at the beginning, but then checked aave and they have `1.5`.

**Review by yivlad — APPROVED** [review]

**Pull request merged**